src/api-server-selector.js

```diff
@@ -361,6 +365,9 @@ export class ApiServerSelector extends EventsTargetMixin(AmfHelperMixin(LitEleme
    * @return {TemplateResult} Custom URI `anypoint-item`
    */
   renderCustomURIOption() {
+    if (this.hideCustom) {
+      return undefined;
```

Comment:
Please, return empty string by default `''` when no template should be rendered. Problems we had with rendering of the API Console in IE11 was caused by returning `undefined` in a template.